modules/objdetect/test/test_qr_utils.hpp

```diff
@@ -0,0 +1,81 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+#include "test_precomp.hpp"
+
+namespace opencv_test {
+
+static inline
+void check_qr(const string& root, const string& name_current_image, const string& config_name,
+                const std::vector<Point>& corners,
+                const std::vector<string>& decoded_info, const int pixels_error,
```

Comment:
`max_pixel_error` or `allowed_pixel_error` is better.

Reply:
fixed